server/channels/app/server.go

```diff
@@ -1238,6 +1241,13 @@ func runConfigCleanupJob(s *Server) {
 	}, time.Hour*24)
 }
 
+func runElasticsearchWorkspaceOptimizationJob(s *Server) {
+	doElasticsearchWorkspaceOptimizationCheck(s)
```

Comment:
Do we need to run it when the server starts? Maybe it's a good idea to just create the recurring task to not add more work to the init process. Thoughts? cc/@isacikgoz

Reply:
Hmm, even though it wouldn't bring much work in general but for users with big tables this could consume some  precious time from the startup. I think we can run this after a while with `CreateRecurringTaskFromNextIntervalTime`.

Reply:
Any suggestions regarding the time? Maybe 10 minutes?

Reply:
Sounds good to me